.github/workflows/ci.yml

```diff
@@ -718,3 +718,64 @@ jobs:
           name: test report pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
           path: testing/trino-product-tests/target/reports/**/testng-results.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+
+  abfs-pt:
+    runs-on: ubuntu-latest
+    name: pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - abfs
+        suite:
+          - suite-abfs
+        jdk:
+          - 11
+    timeout-minutes: 30
+    needs: build-pt
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checkout all commits, as the build result depends on `git describe` equivalent
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 11
+      - name: Product tests artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: product tests and server tarball
+      - name: Fix artifact permissions
+        run: |
+          find . -type f -name \*-executable.jar -exec chmod 0777 {} \;
+      - name: Product Tests
+        env:
+          ABFS_CONTAINER: ${{ secrets.AZURE_ABFS_CONTAINER }}
+          ABFS_ACCOUNT: ${{ secrets.AZURE_ABFS_ACCOUNT }}
+          ABFS_ACCESS_KEY: ${{ secrets.AZURE_ABFS_ACCESSKEY }}
+        if: env.ABFS_CONTAINER != '' && env.ABFS_ACCOUNT != '' && env.ABFS_ACCESS_KEY != ''
```

Comment:
You could set an output variable to the result of this expression in the `build-pt` job and then check it here at the job level to skip the whole job. This would avoid downloading the artifact, which is big, and the job itself will be marked better, improving the UX.

Reply:
@nineinchnick Sorry, I couldn't understand the way to address this comment. Can you share the example?

Reply:
Here are some: https://github.com/actions/runner/issues/520#issuecomment-1077480871

Reply:
Thanks for sharing the example. Verified that PR from forked branch skips the job in #12166.